test/wrapped-tokens/WHBAR.js

```diff
@@ -520,7 +520,7 @@ describe('WHBAR', function () {
   it('WHBAR-023 should revert on value > MaxUint256 for deposit', async function () {
     await expect(
       contract.deposit({ value: OVERFLOW_VALUE })
-    ).to.be.rejectedWith('value cannot exceed MAX_INTEGER');
+    ).to.be.rejectedWith('fields had validation errors');
```

Comment:
The dependabot PR [here](https://github.com/hashgraph/hedera-smart-contracts/commit/a3e15e1b105eb37a44754e6aeffbc5ce0c916672) broke the WHBAR tests, so we're fixing it now.